src/domains/wallet-app/api/npg/v1/_openapi.json.tpl

```diff
@@ -0,0 +1,138 @@
+          "description": "Update Wallet on NPG payment instrument creation",
+          "operationId": "notifyWallet",
+          "parameters": [
+            {
+              "in": "header",
+              "name": "correlation-Id",
+              "schema": {
+                "type": "string",
+                "format": "uuid"
+              },
+              "required": true,
+              "description": "Unique identifier of the request"
+            }
+          ],
+          "requestBody": {
+            "description": "Notify wallet",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "$ref": "#/components/schemas/NotificationRequest"
+                }
+              }
+            },
+            "required": true
+          },
+          "responses": {
+            "204": {
+              "description": "Payment reqeust sent successfully"
+            },
+            "400": {
+              "description": "Invalid input",
+              "content": {
+                "application/json": {
+                  "schema": {
+                    "$ref": "#/components/schemas/ErrorResponse"
+                  }
+                }
+              }
+            },
+            "500": {
+              "description": "Internal server error",
+              "content": {
+                "application/json": {
+                  "schema": {
+                    "$ref": "#/components/schemas/ErrorResponse"
+                  }
+                }
+              },
+            },
+            "405": {
+              "description": "Invalid input"
+            }
+          }
+        }
+      }
+    },
+    "components": {
+      "schemas": {
+        "NotificationRequest": {
+          "properties": {
+            "contractId": {
+              "type": "string",
+              "maxLength": 255,
+              "example": "C2834987"
+            },
+            "status": {
+              "type": "string",
+              "description": "Status of the payment order",
+              "enum": [
+                "OK",
+                "KO"
+              ],
+              "example": "OK"
+            },
+            "securityToken": {
+              "type": "string",
+              "description": "Payment gateway security token used to validate the request",
+              "example": "2f0ea5059b41414ca3744fe672327d85"
+            }
+          }
+        },
+        "ErrorResponse": {
+          "type": "object",
+          "properties": {
+            "errors": {
+              "type": "array",
+              "items": {
+                "$ref": "#/components/schemas/ErrorItem"
+              }
+            }
+          }
+        },
+        "ErrorItem": {
+          "type": "object",
+          "properties": {
+            "code": {
+              "type": "string"
+            },
+            "description": {
+              "type": "string"
+            }
+          }
+        }
+      }
+    }
```

Comment:
Should we use `ProblemJson` object to return problems to the caller as done in ecommerce or also [here](https://github.com/pagopa/pagopa-infra/blob/e1037bda2baeeec28f2c033e06d2e843cce240a1/src/domains/wallet-app/api/wallet-service/v1/_openapi.json.tpl#L334-L374) in the wallet?